ee/migrations/0014_roles_memberships_and_resource_access.py

```diff
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ("posthog", "0278_organization_customer_id"),
+        ("posthog", "0279_recording_playlist_item_model"),
```

Comment:
is this right? The other file shows latest as 280_

Reply:
good catch !